src/authn/auth0.js

```diff
@@ -46,6 +46,12 @@ class Auth0Login {
       });
     });
 
+    // similarly, but use the hosted lock to temporarily allow LDAP
+    router.get('/login-temp', (req, res) => {
+      req.session['auth0-local'] = 1;
+        res.redirect('/auth0/callback');
```

Comment:
nit: indentation

Reply:
ugh, vim and its delayed, incorrect re-indentation..